doc/Type/Supply.pod6

```diff
@@ -370,6 +370,22 @@ Creates a "tail" supply with the same semantics as L<List.tail|/type/List#method
     my $ts = $s.tail(2);
     $ts.tap(&say);           # OUTPUT: «3␤2␤»
 
+=head2 method first
+
+    method first(Supply:D: :$end, |c)
+
+This method is a frontend on the C<grep> method. Its creates a new "head"
```

Comment:
Not clear what you mean here. If it means that it's like getting a supply of the first element that would be returned from grep, say so. Saying it's a frontend for something is like saying you've put a layer over that something, but that it does the same, maybe in a different way.